src/validation.cpp

```diff
@@ -225,6 +224,8 @@ class CChainState {
  */
 RecursiveMutex cs_main;
 
+const CScript WDMO_SCRIPT = CScript() << OP_HASH160 << std::vector<unsigned char>{11, 182, 127, 3, 232, 176, 211, 69, 45, 165, 222, 55, 211, 47, 198, 174, 240, 165, 160, 160} << OP_EQUAL; // TODO: replace with actual wdmo script; use ParseHex
```

Comment:
Validation.cpp is already too big, we should extend it as little as possible. This definition, along with MinerLicences should be moved to some specific directory, e.g. src/policy/ddms

Reply:
Done